packages/reporter/src/std/StdReporter.spec.ts

```diff
@@ -0,0 +1,174 @@
+import { StdReporter } from './StdReporter';
+import { HttpMethod, Issue, Scan, Severity } from '@secbox/scan';
+import { instance, mock, reset, when } from 'ts-mockito';
+
+const highSeverityIssue: Partial<Issue> = {
+  name: 'a',
+  request: {
+    method: HttpMethod.GET,
+    url: 'x'
+  },
+  severity: Severity.HIGH
+};
+
+const mediumSeverityIssue: Partial<Issue> = {
+  name: 'b',
+  request: {
+    method: HttpMethod.GET,
+    url: 'y'
+  },
+  severity: Severity.MEDIUM
+};
+
+const lowSeverityIssue: Partial<Issue> = {
+  name: 'c',
+  request: {
+    method: HttpMethod.GET,
+    url: 'z'
+  },
+  severity: Severity.LOW
+};
+
+const escapeRegex = (s: string) => s.replace(/[.*+?^${}()|[\]\\]/g, '\\$&');
+
+const tableRowRegex = (...columnTexts: string[]) =>
+  columnTexts.map(x => escapeRegex(x)).join('[ │]+');
+
+describe('StdReporter', () => {
+  let reporter!: StdReporter;
+
+  const mockedScan = mock<Scan>();
+
+  beforeEach(() => {
+    jest.spyOn(console, 'error').mockImplementation(() => {
+      /* noop */
+    });
```

Comment:
Is it intended? As far as I know, it should work without a noop implementation. 

```suggestion
    jest.spyOn(console, 'error');
```

Reply:
but then it will be unwanted output in stdout/stderr during tests run?

Reply:
If so, you can leave it as is